proto/store.proto

```diff
@@ -0,0 +1,45 @@
+syntax = "proto3";
+
+option go_package = "github.com/momentohq/client-sdk-go;client_sdk_go";
+option java_multiple_files = true;
+option java_package = "grpc.store";
+option csharp_namespace = "Momento.Protos.Store";
+
+package store;
+
+service Store {
+  rpc Get(_GetRequest) returns (_GetResponse) {}
+  rpc Set(_SetRequest) returns (_SetResponse) {}
+  rpc Delete(_DeleteRequest) returns (_DeleteResponse) {}
+}
+
+message _GetRequest {
+  bytes key = 1;
+  string store_id = 2;
+}
+
+message _GetResponse {
+  oneof result {
+    Miss miss = 1;
+    Hit hit = 2;
+  }
+  message Miss { }
+  message Hit {
+    bytes value = 1;
+  }
+}
+
+message _SetRequest {
+  bytes key = 1;
+  bytes value = 2;
```

Comment:
Following the same naming convention as our cache apis this should be:
```
bytes store_key = 1;
bytes store_value = 2;
```
But I am not sure if we want to name the value as `store_value` or `store_body` as we call it `cache_body` in the cache apis. @cprice404 thoughts?

Reply:
idk where `body` came from, `value` makes sense to me.

Reply:
I think `body` came from `cache_body` in cacheclient.proto

Reply:
sorry, what i'm saying is that I don't know how we ultimately chose `body` in the cache proto. `value` makes more sense to me. And overall I do not feel compelled to make this proto match the cache proto, I'd rather just take everything we've learned and make this one the best it can be.